.github/workflows/go_tests.yml

```diff
@@ -27,5 +27,5 @@ jobs:
       - name: Run Tests
         run: |
           make bin
-          export BENCHMARK_BINARY=$(pwd)/dist/linux/amd64/vault-benchmark
+          export BENCHMARK_BINARY=$(pwd)/dist/linux/amd64/openbao-benchmark
```

Comment:
This `BIN_NAME` differs from `go_tests.yml` and the new one you set in `Makefile` fwiw